README.md

```diff
@@ -10,3 +10,10 @@ Required variables:
 
 Custom static content (HTML files, PDF, etc.) can be deployed
 to ``{{ wifi_login_pages_path }}/static/``.
+
+## Deploy organizations configurations and assets
+
+For deploying organization YAML config files and their related static assets (logo, CSS, etc),
+it is mandatory to paste the `organizations` directory with all the configuration and assets of
+the organizations inside `files/` directory. These files will be uploaded to remote during the
```

Comment:
is this the files/ directory of the playbook or of this ansible role?

We should not ask the user to edit the files directory of this role, but it's ok if the user can accomplish this by placing the org config in the `files/` dir of their playbook.

Reply:
Updated README